From dc11b49da37068243c0a9c09f3728467b05b2079 Mon Sep 17 00:00:00 2001
From: Hongcheng Xie <hongcheng.xie@intel.com>
Date: Mon, 29 Jun 2020 20:35:16 +0800
Subject: [PATCH 7/7] [WA] read from /cmdline

WR and read from /cmdline directly since android-entry write prop into
/cmdline

Change-Id: If236e6d8ce6ff3c476c5aa7b7176b6c72bfcaba4
Signed-off-by: Hongcheng Xie <hongcheng.xie@intel.com>
---
 init/util.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/init/util.cpp b/init/util.cpp
index 4e9950719..a2ea46ea5 100644
--- a/init/util.cpp
+++ b/init/util.cpp
@@ -246,7 +246,8 @@ int wait_for_file(const char* filename, std::chrono::nanoseconds timeout) {
 
 void ImportKernelCmdline(const std::function<void(const std::string&, const std::string&)>& fn) {
     std::string cmdline;
-    android::base::ReadFileToString("/proc/cmdline", &cmdline);
+    //android::base::ReadFileToString("/proc/cmdline", &cmdline);
+    android::base::ReadFileToString("/cmdline", &cmdline);
 
     for (const auto& entry : android::base::Split(android::base::Trim(cmdline), " ")) {
         std::vector<std::string> pieces = android::base::Split(entry, "=");
-- 
2.17.1

